rocketpool-cli/commands/pdao/proposals.go

```diff
@@ -86,6 +88,10 @@ func getProposals(c *cli.Context, stateFilter string) error {
 
 		// Proposals
 		for _, proposal := range proposals {
+			if len(proposal.Message) > 200 {
+				proposal.Message = proposal.Message[:200]
```

Comment:
same comment as above, but since this happens in two places, maybe create a function that:
1. truncates to 197
2. append "..."
3. calls utilStrings.Sanitize